dashboard/lib/services/user/password_resetter.rb

```diff
@@ -0,0 +1,58 @@
+module Services
+  module User
+    class PasswordResetter < Services::Base
+      attr_reader :email, :user
+
+      def initialize(email:)
+        @email = email
+        @user = nil
+      end
+
+      def call
+        if email.blank?
+          @user = ::User.new
+          user.errors.add :email, ::I18n.t('activerecord.errors.messages.blank')
+          return user
+        end
+        # We are no longer sending an email to parents, so grab the first user we find
+        # (a user with an Email auth option first, otherwise any user that has that email)
+        @user = ::User.find_by_email_or_hashed_email(email)
+        if user.nil?
+          not_found_user = ::User.new(email: email)
```

Comment:
Since we aren't reusing this variable we could just return `::User.new(email: email)` directly on line 27